core-aam/aria-checked_value_changes-manual.html

```diff
@@ -58,7 +58,7 @@
                   "event",
                   "type",
                   "is",
-                  "TogglePattern.ToggleStateProperty"
+                  "PropertyChangedEvent"
```

Comment:
Most of the event entries for UIA also list which attributes change, but I wasn't sure whether to test that in these tests. For example: https://w3c.github.io/core-aam/#event-aria-checked